.github/workflows/ci.yml

```
@@ -108,6 +117,8 @@ jobs:
           DB_DATABASE: test
           DB_USERNAME: root
           DB_PASSWORD: password
+          REDIS_HOST: 127.0.0.1
+          REDIS_PORT: ${{ job.services.redis.ports[6379] }}
```

actionlint (1.7.7)

property access of object must be type of string but got "number"

(expression)